### PR TITLE
Completed classmethods for Item.py (database functions)

### LINKED ISSUE #8: Database functions to work with Forms
need to integrate it with OOP

- [x] research which is better: UUID or ObjectID with mongo and python
- [x] get_all()
- [x] get_item_by_id()
- [x] get_item_by_category()

FYI: Database functions are defined in the classes

### PR DESCRIPTION
**Pull request by elenirotsides**
Right now, I've got a barebones setup going with a dummy addItem() function. Marking this as a draft for now!﻿

### CODE REVIEW

File: src/e4stypes/item.py [hunk 1]
```diff
@@ -1,15 +1,29 @@
 from decimal import Decimal
 from typing import List
+from bson import ObjectId
+import pymongo
```
Comment (dcarpenter31) [inline src/e4stypes/item.py:4]: did you `pip install` these? if so, do you wanna add them to `requirements.txt`?

Comment (elenirotsides) [reply src/e4stypes/item.py:4]: For line 3, no, its included in pymongo (almost positive)! For line 4, I can't remember! When I did `pip install pymongo`, it said it was already there. I'll add it though, it wouldn't hurt!

**Comment by elenirotsides** [comment]
In order to test, you need to make sure you have mongo installed, see https://docs.mongodb.com/manual/administration/install-community/ for details. I also recommend installing mongo Compass (https://docs.mongodb.com/compass/master/install/), which is basically a GUI for mongo. I prefer compass over the traditional way of interacting via the command line. Please let me know if there are any questions!

**Comment by dcarpenter31** [comment]
would you be able to add instructions for this in the `README` so that we can reference it from there?

**Comment by elenirotsides** [comment]
Ah yes, that would be much much better!

**Comment by elenirotsides** [comment]
@carpenterd777 Added!

File: README.md [hunk 2]
```diff
@@ -39,3 +39,17 @@ If you have python 2 and python 3 installed, python 2 will most likely be the de
...
+
+By default, Mongo will run on port 27017, so don't change that because that's what the code references right now.
+
+When you open MongoDB Compass, it'll ask you to enter the `Connection String`. Here are what my settings look like:
+
+![Mongo settings](mongo.png)
+
+FYI: I don't use my local instance of MongoDB for any intense dev work, which is why I don't have anything selected for the Authentication field.
+
+**I recommend saving your connection string as a favorite in Compass, so you don't have to keep typing stuff in each time you want to connect to the databse**
```
Comment (dcarpenter31) [inline README.md]:
```suggestion
**I recommend saving your connection string as a favorite in Compass, so you don't have to keep typing stuff in each time you want to connect to the database**
```

**Review by dcarpenter31 — APPROVED** [review]
looks good!

**Pull request merged**